tests/test_screenshot.py

```diff
@@ -142,6 +144,32 @@ def test_posts_multiple_screenshots_to_the_local_percy_server(self):
         self.assertRegex(s1['environment_info'][0], r'appium/\d+')
         self.assertRegex(s1['environment_info'][1], r'python/\d+')
 
+    @patch.object(Metadata, 'session_id', PropertyMock(return_value='unique_session_id'))
+    @patch.object(GenericProvider, '_write_screenshot', MagicMock(return_value='path-to-png-file'))
+    def test_ignore_region_screenshots_to_the_local_percy_server(self):
+        mock_healthcheck()
+        mock_screenshot()
+
+        # mock element
+        mock_element = MagicMock()
+        mock_element.location = {"x": 10, "y": 20}
+        mock_element.size = {"width": 200, "height": 400}
+
+        xpaths = ["//path/to/element"]
+        self.mock_webdriver.find_element.return_value = mock_element
+
+        percy_screenshot(self.mock_webdriver, 'screenshot 1', ignore_regions_xpaths = xpaths)
+
+        self.assertEqual(httpretty.last_request().path, '/percy/comparison')
+
+        s1 = httpretty.latest_requests()[1].parsed_body
+
+        # self.assertEqual(s1['name'], 'screenshot 1')
+        self.assertRegex(s1['client_info'], r'percy-appium-app/\d+')
```

Comment:
where is the assertion that ignore element data is passed with expected value ?

Reply:
done. it also covers the above incorrect args location.